vsintegration/src/FSharp.Editor/CodeFixes/CodeFixHelpers.fs

```diff
@@ -70,6 +75,38 @@ module internal CodeFixHelpers =
             name
         )
 
+    let getUnusedSymbol textSpan (document: Document) codeFixName =
```

Comment:
This stuff used to be in the `RenameUsusedValue` file. It's now here, because it's used by two code fixes. It also takes an extra parameter for the operation name.

This should be refactored - but preferably in a followup.